src/mrb_capability.c

```diff
@@ -266,6 +294,87 @@ mrb_value mrb_cap_getgid(mrb_state *mrb, mrb_value self)
     return mrb_fixnum_value((mrb_int)getgid());
 }
 
+mrb_value mrb_file_cap_init(mrb_state *mrb, mrb_value self)
+{
+    char *path;
+    mrb_file_cap_context *cap_ctx = (mrb_file_cap_context *)mrb_malloc(mrb, sizeof(mrb_file_cap_context));
+
+    mrb_get_args(mrb, "z", &path);
+    cap_ctx->path = path;
+    cap_ctx->cap = cap_get_file(path);
+
+    mrb_iv_set(mrb
+        , self
+        , mrb_intern_cstr(mrb, "mrb_file_cap_context")
+        , mrb_obj_value(Data_Wrap_Struct(mrb
+            , mrb->object_class
+            , &mrb_file_cap_context_type
+            , (void *)cap_ctx)
+        )
+    );
+
+    return self;
+}
+
+mrb_value mrb_file_cap_path(mrb_state *mrb, mrb_value self)
+{
+    mrb_file_cap_context *cap_ctx = mrb_file_cap_get_context(mrb, self, "mrb_file_cap_context");
+    return mrb_str_new_cstr(mrb, cap_ctx->path);
+}
+
+mrb_value mrb_file_cap_set_file(mrb_state *mrb, mrb_value self)
+{
+    mrb_value ary;
+    mrb_int identify;
+    int i, ret;
+    mrb_file_cap_context *cap_ctx = mrb_file_cap_get_context(mrb, self, "mrb_file_cap_context");
```

Comment:
`mrb_file_cap_context *cap_ctx ` name is same as `mrb_cap_context *cap_ctx `. So you should change the name of `mrb_file_cap_context *cap_ctx ` to `mrb_file_cap_context *file_cap_ctx ` for readability.